src/textual/widgets/_placeholder.py

```diff
@@ -8,9 +10,11 @@
 from .. import events
 from ..css._error_tools import friendly_list
 from ..reactive import Reactive, reactive
-from ..widget import RenderResult, Widget
+from ..widget import Widget
 
 PlaceholderVariant = Literal["default", "size", "text"]
+"""The different variants of placeholder."""
```

Comment:
What do you think of moving the variant table from `Placeholder` up to here?

Reply:
Missed this. Yeah, I was debating that too.